doc/configuration.rst

```diff
@@ -1045,6 +1045,18 @@ metric_namespace
   Optional prefix to add to the beginning of every metric sent to Datadog.
   Default value is "luigi".
 
+[prometheus]
+---------
+
+use_task_family
```

Comment:
The naming of this config value isn't very self-documenting. Is there an adjustment you can make to the name of this option such to be clearer to the user without requiring consulting documentation for its intent?

Reply:
same goes for `task_parameters`.

Both of these options refer to bucket labels, but neither name give this context.

A quick glance at the prometheus client for `Counter`, looks like the labels that are generated are passed as `labelnames` https://github.com/prometheus/client_python/blob/5926a7c0ddb8755a5e504e6f077e081f5f4c7674/prometheus_client/metrics.py#L105

Reply:
II changed names to: use_task_family_in_labels and task_parameters_to_use_in_labels. Hope that will make fields meaning more clear.